ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1627,3 +1627,17 @@ def igesin(self, fname="", ext="", **kwargs):
             out = super().igesin(basename, **kwargs)
 
         return out
+
+    @wraps(_MapdlCore.cmatrix)
+    def cmatrix(self, symfac="", condname="", numcond="", grndkey="",
+                capname="", **kwargs):
+        """Wrap the CMATRIX command to update correctly the `last_response` command. """ 
+
+        """The CMATRIX command needs to run in non-interactive mode"""
+        empty_output = super().cmatrix(self, symfac, condname, numcond, grndkey, capname, **kwargs)
+       
+        # Reading output
+        output = self._download_as_raw('cmatrix.out').decode()
+        self._response = output 
+
+        return output 
```

Comment:
```suggestion
        self._response = self._download_as_raw('cmatrix.out').decode()
        return self._response
```
No reason to use `output` as a parameter.

Reply:
You are right. Fixed now.